eclair-core/src/main/scala/fr/acinq/eclair/wire/LightningMessageCodecs.scala

```diff
@@ -28,6 +28,8 @@ import scodec.{Attempt, Codec}
  */
 object LightningMessageCodecs {
 
+  final val STANDARD_UNKNOWN_MESSAGE_TAG = 42263
```

Comment:
We should not have to hardcode something plugin-specific like that inside eclair-core.
See other comments below.